src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -349,6 +349,11 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 }
 
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
+	// If an item does not have a title, try using the feed's title instead
+	if item.Title == "" {
+		item.Title = feed.Title
```

Comment:
This is mutating the passed in object, which probably isn't bad in this case but in general I'd try to avoid it if possible as it can lead to hard to debug bugs. Possibly define a new variable `itemTitle` for example?